content/blog/azure-native-v3/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: "Azure Native V3: A Leaner, Faster SDK for Microsoft Azure"
+date: 2025-04-03
+draft: false
+meta_desc: Azure Native V3 delivers 75% smaller SDK size, updated API versions, and flexible options for working with explicit API versions in Microsoft Azure.
+meta_image: meta.png
+authors:
+    - thomas-kappler
+    - meagan-cojocar
+
+tags:
+    - azure-native
+    - azure
+    - features
+    - releases
+
+social:
+    twitter:
+    linkedin: 
+---
+
+Today we're excited to announce the release of Pulumi Azure Native V3 - a major leap forward for the most comprehensive infrastructure as code solution for Microsoft Azure. This release delivers a remarkable 75% reduction in SDK size while maintaining our complete coverage of the Azure ecosystem.
+
+## The Undisputed Leader for Azure Infrastructure as Code
+
+Pulumi Azure Native provides direct access to the [Azure Resource Manager API](https://learn.microsoft.com/en-us/rest/api/resources/), making it hands-down the best infrastructure as code solution for Microsoft Azure. Unlike traditional providers that abstract away cloud APIs, Azure Native gives you more than **2x the resource and property coverage** compared to Terraform by directly mapping to Azure's native REST API. This means:
+
+- **Complete API coverage**: 100% of Azure resources available in Azure Resource Manager
+- **Same-day access to new Azure features**: No waiting for provider updates
+- **Advanced property support**: Access to all resource properties, not just a subset
+- **Latest API versions**: Always up-to-date with the newest stable Azure APIs
+
+<!--more-->
+
+And now with V3's dramatic 75% SDK size reduction, we've eliminated the only potential drawback - making our Azure Native provider both the most comprehensive AND the most performant option for managing Azure infrastructure.
+
+![Resource and Property Coverage Comparison](https://www.pulumi.com/images/blog/azure-native-coverage-comparison.png)
+
+These improvements came about through our [RFC process](https://github.com/pulumi/pulumi-azure-native/issues/4004), where we explored various options to balance SDK size with the flexibility needed by advanced users.
+
+## A Slimmer, More Efficient SDK
+
+The most significant improvement in Azure Native V3 is the dramatic 75% reduction in SDK size across all languages. By optimizing the SDK to include only the most commonly used API versions, we've solved one of the most frequent pain points reported by our users:
```

Comment:
> the most commonly used API versions

That could be interpreted in different ways.

- There's exactly one API version per resource. People might worry that we dropped long-tail resources.
- We didn't look at resource usage among Pulumi users, so "popular" is a bit off.

Rather, we have an algorithm that weighs new-ness, stable over preview, and a few other factors.

Reply:
do we have docs on how the algorithm works? and a policy for when we update the default?